### PR TITLE
Allow custom options for Raven

### PR DESCRIPTION
**Pull request by aivus**
This PR allows to configure custom options for Raven.

I'm using it to enable `autoBreadcrumbs` for debug and receive additional breadcrumbs to the Sentry in case of any issues with the app.

### CODE REVIEW

**Comment by aivus** [comment]
@RobinBol wdyt?

File: lib/Log.js [hunk 1]
```diff
@@ -4,21 +4,28 @@ const HomeyModule = require('homey');
...
+   *
    * @example
    * class MyApp extends Homey.App {
    *   onInit() {
    *     this.homeyLog = new Log({ homey: this.homey });
    *   }
    * }
    */
-  constructor({ homey }) {
+  constructor({ homey , options}) {
```
Comment (aivus) [inline lib/Log.js:28]: Maybe event `options = {}` need to have default value

**Comment by RobinBol** [comment]
Not opposed to merging this :) Don't really have a usecase for it myself, but seems like a good addition to at least enable options this way. What kind of default value do you consider for `options = {}`? Empty object seems fine by me.

**Comment by aivus** [comment]
I'm personally using `{ autoBreadcrumbs: true }` because [Sentry Breadcrumbs](https://docs.sentry.io/product/issues/issue-details/breadcrumbs/) are providing additional data to the Sentry OOTB:
Screenshot 2021-12-28 at 21 10 55

So, I would suggest to have something like this, but empty object s also ok for me as I'll be able to pass required options.

@RobinBol wdyt, should we pass `autoBreadcrumbs` or just leave it empty?

**Review by RobinBol — APPROVED** [review]

**Comment by RobinBol** [comment]
> @RobinBol wdyt, should we pass `autoBreadcrumbs` or just leave it empty?

I'll leave it as is for now and publish a new release, thanks!

**Pull request merged**